kustomize/apiserver/clusterpedia_apiserver.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
```

Comment:
Last issue, this file is also generated by `kubectl kustomie`

Reply:
Missed that . will update